FightPandemics/Views/Components/Overlays/OverlayDemo.swift

```diff
@@ -0,0 +1,53 @@
+//
+//  OverlayDemo.swift
+//  FightPandemics
+//
+//  Created by Zizhou Wang on 9/24/20.
+//  Copyright © 2020 Scott. All rights reserved.
+//
+
+import SwiftUI
+
+struct OverlayDemo: View {
+    @State var showOverlay = false
+    @State var dragTranslate = CGSize.zero
```

Comment:
...private

Reply:
Yes, these need to be changed to private, great spot Luko